api/api-doc/storage_service.json

```diff
@@ -2324,6 +2359,103 @@
                "description":"The endpoint details"
             }
          }
+      },
+      "map_entry":{
+        "id":"map_entry",
+        "properties":{
+            "key":{
+                "type":"string"
+            },
+            "value":{
+                "type":"string"
+            }
+        }
+      },     
+      "named_maps":{
+        "id":"named_maps",
+        "properties":{
+            "group":{
+                "type":"string"
+            },
+            "attributes":{
+                "type":"array",
+                "items":{
+                    "$ref":"map_entry"
+                }
+            }
+        }
+      },
+      "sstable":{
+        "id":"sstable",
+        "properties":{
+            "size":{
+               "type":"long",
+               "description":"Total size in bytes on"
```

Comment:
didn't understand the description

Reply:
Must have slipped with clipboard or something. Should read "of sstable".